doc/devel/documenting_mpl.rst

```diff
@@ -818,12 +816,28 @@ code will also appear in interactive docstrings.
 Writing examples and tutorials
 ==============================
 
-Examples and tutorials are python scripts that are run by `Sphinx Gallery`_
-to create a gallery of images in the :file:`/doc/gallery` and
-:file:`/doc/tutorials` directories respectively.  To exclude an example
+Examples and tutorials are python scripts that are run by `Sphinx Gallery`_.
+Sphinx Gallery finds ``*.py`` files in source directories and runs the files to
+create images and narrative that are embedded in ``*.rst`` files in a build
+location of the :file:`doc/` directory.  Files in the build location should not
+be directly edited as they will be overwritten by Sphinx gallery. Currently
+Matplotlib has four galleries as follows:
+
+===============================  ==========================
+Source location                  Build location
+===============================  ==========================
+:file:`galleries/plot_types`     :file:`doc/plot_types`
+:file:`galleries/examples`       :file:`doc/gallery`
+:file:`galleries/tutorials`      :file:`doc/tutorials`
+:file:`galleries/users_explain`  :file:`doc/users/explain`
+===============================  ==========================
+
+The first three are traditional galleries, :file:`galleries/users_explain` is a
+mixed gallery where some of the files are raw ``*.rst`` files rather than
+``*.py`` files; Sphinx Gallery just copies those from the location to the build
+location (see :ref:`raw_restructured_gallery`, below). To exclude an example
 from having an plot generated insert "sgskip" somewhere in the filename.
```

Comment:
There may be a sentence part missing here?

Reply:
Maybe it's correct English, but I agree it's not very easy to parse.  I'll take another swing at it.

Reply:
OK, modified to be a little more clear (I hope)